tyk-pro/templates/bootstrap-role.yml

```diff
@@ -29,6 +29,7 @@ rules:
   - jobs
   verbs:
   - delete
+  - list
```

Comment:
Just an idea - What do you think about listing required roles in https://github.com/TykTechnologies/tyk-k8s-bootstrap?

Reply:
Done